### PR TITLE
feat-typescript: add chr

### PR DESCRIPTION
**Pull request by afman42**
- Programming langugage: typescript
- Function(s) that I'm implementing: chr
- Issue number: 

- [x] I have created unit tests to support my implementation
- [x] I have created in-code documentation (or docstring) to document my code
- [ ] (Optional) I have updated the README.md file to document the function I'm implementing

### CODE REVIEW

File: typescript/src/string/str_repeat.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/**
+ * str_repeat — Repeat a string
+ * @param str - The string to be repeated.
+ * @param times - Number of time the `str` string should be repeated. `times` has to be greater than or equal to 0. If the `times` is set to 0, the function will return an empty string.
```
Comment (aldy505) [inline typescript/src/string/str_repeat.ts:4]: tambahin tes kalo times nya itu negatif

File: typescript/src/string/chr.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/**
+ * chr — Generate a single-byte string from a number.
+ * @param ascii - An integer between 0 and 255.
```
Comment (aldy505) [inline typescript/src/string/chr.ts:3]: tambahin tes kalo ascii number ini < 0 dan > 255

**Review by aldy505 — CHANGES_REQUESTED** [review]

File: typescript/src/string/str_repeat.ts (hunk 1 above)
Comment (afman42) [reply typescript/src/string/str_repeat.ts:4]: ini udah di merged sama @elianiva

File: typescript/src/string/chr.ts (hunk 2 above)
Comment (afman42) [reply typescript/src/string/chr.ts:3]: oke pak

File: typescript/src/string/str_repeat.ts (hunk 1 above)
Comment (elianiva) [reply typescript/src/string/str_repeat.ts:4]: sekalian tambahin di PR ini

Comment (afman42) [reply typescript/src/string/str_repeat.ts:4]: oke kak

File: typescript/src/string/chr.ts (hunk 2 above)
Comment (afman42) [reply typescript/src/string/chr.ts:3]: silakan cek lagi pak @aldy505

**Review by aldy505 — APPROVED** [review]

**Pull request merged**